### PR TITLE
Fix: CI examples not throwing error from some commands

### PR DESCRIPTION
**Pull request by selmanozleyen**
Here is an example case where it throws error but the CI still doesn't fail. This PR will fix that issue

image

- The script run is now a separate file
- I also added tutorials folder to the list of notebooks to execute
- added the dependencies to add the tutorials folder

It is ready for review. Once the notebooks are updated it is ready for merge imo

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -302,12 +302,13 @@ deps =
     jupytext
     nbconvert
     leidenalg
+    squidpy
+    pot
```
Comment (MUCDK) [inline pyproject.toml:306]: why do we need this?

**Comment by MUCDK** [comment]
Amazing, thanks @selmanozleyen ! @ArinaDanilina , please update the notebooks accordingly!

File: pyproject.toml (hunk 1 above)
Comment (giovp) [reply pyproject.toml:306]: exactly, installation of squidpy is also in the [spatial] optional requirements

**Comment by giovp** [comment]
I think the tutorial folder is a bit excessive, and examples should be fine?

File: pyproject.toml (hunk 1 above)
Comment (selmanozleyen) [reply pyproject.toml:306]: For the tutorials folder. If we don't need to check them then I can remove.

**Comment by MUCDK** [comment]
I think tutorials is too much, especially once we replace them with larger data. Then CI would just take for too long. Let's rather have them checked manually regularly

**Review by MUCDK — APPROVED** [review]
LGTM, thanks!

**Pull request merged**